RepairsListener/Infrastructure/AddRepairsContractsToNewAssetObject.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RepairsListener.Infrastructure
+{
+    // The same class is also defined in Asset API to ensure the information sent and received is in the same format
+    public class AddRepairsContractsToNewAssetObject
+    {
+        public Guid EntityId { get; set; }
+        public string PropRef { get; set; }
+        public bool AddRepairsContracts { get; set; }
+    }
+}
```

Comment:
Had a look at other listeners to see how they treated the data from the SNS message and there's not one common way to handle these. In some cases what's inside the message can be converted directly into an existing domain class (like into an Asset for example) or in some other cases the data is converted into a generic object. 

Some examples:

- ![image](https://github.com/LBHackney-IT/repairs-listener/assets/70756861/02cdc5fa-e97f-437d-8b60-29f1b17abcf0)

- ![image](https://github.com/LBHackney-IT/repairs-listener/assets/70756861/297832b5-2451-4132-9d91-6850bd251a5f)

Had a look in Hackney Core and I changed my mind, I don't think that'd be the right place to move this class. Ideally it should go into a Repairs Shared, which we don't have, so for now the same class is here with a duplicate in Asset API

Reply:
it's fine here for now